src/Sylius/Component/Core/Model/OrderInterface.php

```diff
@@ -132,6 +132,8 @@ public function setCreatedByGuest(bool $createdByGuest): void;
      * @psalm-return Collection<array-key, OrderItemInterface>
      *
      * @psalm-suppress ImplementedReturnTypeMismatch
+     *
+     * @phpstan-ignore-next-line
```

Comment:
As above, why ignore?

Reply:
```
Return type (Doctrine\Common\Collections\Collection<(int|string), Sylius\Component\Core\Model\OrderItemInterface>) of method
         Sylius\Component\Core\Model\OrderInterface::getItems() should be compatible with return type (Doctrine\Common\Collections\Collection<(int|string),
         Sylius\Component\Order\Model\OrderItemInterface>) of method Sylius\Component\Order\Model\OrderInterface::getItems()
```
Obviously here it is right to return a list of core order items, it sounds like a PHPStan bug...